pkgs/applications/office/qnotero/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, python3, python3Packages, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "qnotero-${version}";
+  version = "1.0.0";
+  src = fetchFromGitHub {
+    owner = "smathot";
+    repo = "qnotero";
+    rev = "release/1.0.0";
```

Comment:
you could embed the version here  like `"release/${version}"` to avoid repetition in the future.